### PR TITLE
July workshop

### PR DESCRIPTION
**Pull request by teresa-m**
It would be great to get some help on what could go wrong in my current state of the event page. Thanks a lot!

### CODE REVIEW

File: events/2024-07-22-freiburg-july.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+- Galaxy Introduction and Quality Control
+
+- ChIP-seq Analysis
+
+- RNA-seq Analysis
+
+- Variant Calling
+
+- Microbiome Analysis
```
Comment (bgruening) [inline events/2024-07-22-freiburg-july.md:93]:
```suggestion
- Galaxy Introduction and Quality Control
- ChIP-seq Analysis
- RNA-seq Analysis
- Variant Calling
- Microbiome Analysis
```

File: events/2024-07-22-freiburg-july.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      - type: custom
+        name: "Thusday "
+        time: "09:15 - 17:00"
+        description: |
+          Variant Calling/Exome Seqencing
+      - type: custom
+        name: "Friday "
+        time: "09:15 - 17:00"
+        description: |
+          Metagenomics
```
Comment (bgruening) [inline events/2024-07-22-freiburg-july.md:74]:
```suggestion
        description: Introduction and Quality control
      - type: custom
        name: "Tuesday "
        time: "09:15 - 17:00"
        description: ChIP-Sequencing
      - type: custom
        name: "Wednesday "
        time: "09:15 - 17:00"
        description: RNA-Sequencing
      - type: custom
        name: "Thursday "
        time: "09:15 - 17:00"
        description: Variant Calling/Exome Sequencing
      - type: custom
        name: "Friday "
        time: "09:15 - 17:00"
        description: Metagenomics
```

**Review by bgruening — DISMISSED** [review]
I guess we will reuse this template a lot, so maybe we can make it a bit more compact?

**Comment by teresa-m** [comment]
I would be happy now with the current state of the event. 
@erxleben @bgruening: please let me know if something is missing or if it should be more compact.

**Review by bgruening — APPROVED** [review]
Thanks, lets get this online.

**Pull request merged**